keps/sig-cloud-provider/991-cloud-controller-migration/README.md

```diff
@@ -102,14 +103,14 @@ _primary_ and N configurable _secondary_ (a.k.a migration) leader election locks
 The primary lock represents the current leader election resource lock in the KCM and the CCM. The set of
 secondary locks are defined by the cloud provider and run in parallel to the primary locks. For a migration
 lock defined by the cloud provider, the cloud provider also determines the set of controllers run within the
-migration lock and the controller manager it should run in - either the CCM or the KCM.
+migration lock and the controller manager it will run in - either the CCM or the KCM.
 
 The properties of the migration lock are:
   * must have a unique name
   * the set of controllers in the lock is immutable.
-  * no two migration locks should have overlapping controllers
+  * no two migration locks will have overlapping controllers
```

Comment:
This is actually a should.

Reply:
Probably bad things will happen. However we have no way to prevent someone from doing this. Since its outside our control, will isn't appropriate because we cannot ensure this is true. I'd by "two migration controllers must not have". Personally though I would leave it at will as the strongest I can say is probably bad things will happen. There may be an edge case we haven't thought of (Eg. some weird round robin controller) where this would be ok.

Reply:
Noted. I will be sure to take special care of this case.